### DISCUSSION TITLE
132

### DISCUSSION THREAD

**Discussion by giscus** [Shane-Park/markdownBlog#163] [Comments]
# 132

https://github.com/CodeSeven/toastr toastr is a Javascript library for non-blocking notifications. jQuery is required. The goal is to create a simple core library that can be customized and extended. toastr 이란 ? toastr 은 non-blocking 알람을 띄우는 Javasript 라이브러리 입니다. jQuery 가 필요합니다 여러가지 다양한 옵션을 줄 수 있지만 간단하게 사용 할 수 있도록 쉽게 설명을 해 드리겠습니다. 테스트 먼저 위의 기본적인 html 코드를 실행해보면 정말 기본적으로, &#39;테스트&#39; 버튼을 눌렀을때 hi ..

https://shanepark.tistory.com/132

**Comment by Ifcompany-kor** [comment]
감사합니다
